server/src/main/java/org/opensearch/rest/action/document/RestBulkAction.java

```diff
@@ -97,6 +97,8 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         Boolean defaultRequireAlias = request.paramAsBoolean(DocWriteRequest.REQUIRE_ALIAS, null);
         bulkRequest.timeout(request.paramAsTime("timeout", BulkShardRequest.DEFAULT_TIMEOUT));
         bulkRequest.setRefreshPolicy(request.param("refresh"));
+        bulkRequest.batchIngestionOption(request.param("batch_ingestion_option", ""));
```

Comment:
This ends up delegating the call to `from()` or my proposed `valueOf()`, both of which will throw an `IllegalArgumentException` on the default empty string here.

Reply:
We set it to "DISABLED" for null or blank